### PR TITLE
Add blog post

### PR DESCRIPTION
**Pull request by carabaestlein**

### CODE REVIEW

File: _posts/2020-07-21-react-native-tracker-0-1-0-released/index.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+- Automatic tracking: The tracker includes automatic tracking of the mobile context, screen context and session context, as well as installs and lifecycle events. All are configurable on tracker initialisation.
+
+- Standard events: Screen View, Page View, and Structured events are available.
+
+- Custom tracking: Custom Self-Describing events and contexts are available.
+
+- Setting subject data: The subject can be set using the `setSubjectData()` method.
+
+
+<h2 id="changes">2. Changes since the alpha release</h2>
```
Comment (colmsnowplow) [inline _posts/2020-07-21-react-native-tracker-0-1-0-released/index.md:38]: Should this be included in the blog post at all? 

I put it in the discourse post as the audience for it includes those that have been on the alpha. But as a 'first release' announcement it feels weird to talk about the alpha.

Comment (carabaestlein) [reply _posts/2020-07-21-react-native-tracker-0-1-0-released/index.md:38]: Good point, let me rephrase that bit slightly!

Comment (carabaestlein) [reply _posts/2020-07-21-react-native-tracker-0-1-0-released/index.md]: @colmsnowplow what do you think? I have left it in but moved it towards the end and also rephrased it slightly...

Comment (colmsnowplow) [reply _posts/2020-07-21-react-native-tracker-0-1-0-released/index.md]: Love it.

File: _posts/2020-07-21-react-native-tracker-0-1-0-released/index.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+```
+
+For a full list of all the tracker initialization options, including automatic sessionization and lifecycle events, please see the [Snowplow React Native Tracker documentation](https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/react-native-tracker/#configuration). 
+
+
+<h2 id="feedback">3. Feedback and next steps</h2>
+
+As this is an early release tracker, we welcome feedback and and of course contributions! Please feel free to open topics with ideas in our [Discourse forum](https://discourse.snowplowanalytics.com/), and add issues and bug reports to the [repository](https://github.com/snowplow-incubator/snowplow-react-native-tracker/issues).
+
+The next release will include the introduction of automated testing. PRs are still welcome, but due to the time investment required for thorough manual testing, they may not be reviewed and merged until we have those tests in place. If in doubt, open an issue or a discourse post to ask about your contribution. Important contributions, especially bugfixes, will be prioritised.
```
Comment (colmsnowplow) [inline _posts/2020-07-21-react-native-tracker-0-1-0-released/index.md:67]: Perhaps we leave this out, but link to the CONTRIBUTING.md file instead (which mentions this). Doesn't feel like we need to publicise what we didn't get round to doing.

Comment (carabaestlein) [reply _posts/2020-07-21-react-native-tracker-0-1-0-released/index.md]: Good catch, have updated!
